src/cpp/session/SessionModuleContext.cpp

```diff
@@ -376,6 +376,29 @@ SEXP rs_markdownToHTML(SEXP contentSEXP)
    return r::sexp::create(htmlContent, &rProtect);
 }
 
+SEXP rs_setPersistentValue(SEXP nameSEXP, SEXP valueSEXP)
+{
+   std::string name = r::sexp::safeAsString(nameSEXP);
+   std::string value = r::sexp::safeAsString(valueSEXP);
+   persistentState().settings().set(name, value);
```

Comment:
Should keys from rstudioapi be namespaced in some way so that they're guaranteed not to conflict with keys used internally by RStudio?